src/api/payment-requests/payment-requests.md

```diff
@@ -904,14 +904,15 @@ Voiding a payment request will cancel the request and trigger any refunds if nec
+| Status |                   Code                   |                                                                                                                     Description                                                                                                                     |
+| :----- | :--------------------------------------- | :-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| 403    | {% break _ VOID_WINDOW_EXCEEDED %}       | The void window is closed 24 hours after the request's `createdAt`. After the void window has closed if the request is paid, the [Refund][] endpoint can be used to reverse the payment.                                                            |
+| 403    | {% break _ ALREADY_REFUNDED %}           | The payment request has already been refunded.                                                                                                                                                                                                      |
+| 403    | {% break _ REPEAT_REFERENCE %}           | A refund has already been requested with the same external reference. Refunding the payment request twice with the same external reference is not allowed. If the amount of the refund is the same we assume it is a repeat request and return 200. |
+| 403    | {% break _ INACTIVE_ASSET %}             | The asset is not refundable. It may have been disabled, expired, or already refunded.                                                                                                                                                               |
+| 403    | {% break _ REFUND_NOT_SUPPORTED %}       | The asset type does not support refunds.                                                                                                                                                                                                            |
+| 403    | {% break _ REQUEST_EXPIRED %}            | The payment request has expired.                                                                                                                                                                                                                    |
+| 403    | {% break _ PRE_AUTH_ALREADY_CONFIRMED %} | The pre auth payment request already has confirmations. We recommend using refund to reverse the transaction.                                                                                                                                                        | 
```

Comment:
Lets get rid of "We recommend" and use active tone.

Lets get rid of the "Void this confirmation" from elsewhere in the document.

Can we use capitals on the nouns here?